src/PrestaShopBundle/Resources/config/routing/admin/configure/shop_parameters/meta.yml

```diff
@@ -33,7 +33,7 @@ admin_metas_edit:
       id_meta: metaId
 
 admin_metas_delete:
-  path: /delete/{metaId}
+  path: /{metaId}/delete/
   methods: DELETE
```

Comment:
@matks i know it's not related to your PR, but do you think we should get rid of `DELETE` method? it's not an API, i think `GET` & `POST` are enough, wdyt?

Reply:
sorry, @toutantic want us to use the right HTTP verbs when we can.

Reply:
I agree with that, but only in context of API. When working with forms, the only right methods are POST/GET. Even in this case, it's POST method that is overridden by framework. That's why I don't see much gain here, it only adds confusion to developers that are not familiar with this feature.

Reply:
If this uri is called by a form in ajax, it can be considered as an API (because of remote call and because we should call API when using this kind of resource).
Agreed with them to use the right HTTP verb!